UsageExamples/OoplesIndicatorLibraryExample.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Alpaca.Markets;
+using OoplesFinance.StockIndicators.Models;
+using static OoplesFinance.StockIndicators.Calculations;
+
+namespace UsageExamples
+{
+    public class OoplesIndicatorLibraryExample
```

Comment:
Make this class `internal sealed` as all other examples classes in this project.